dev/dev/devDeployment.yaml

```diff
@@ -49,7 +49,9 @@ datarepo-api:
     RBS_POOL_ID: datarepo_v3
     RBS_INSTANCE_URL: https://buffer.dsde-dev.broadinstitute.org
     SAM_OPERATION_TIMEOUT_SECONDS: 30
-    TERRA_COMMON_TRACING_STACKDRIVER_EXPORT_ENABLED: false
+    TERRA_COMMON_STAIRWAY_TRACINGENABLED: true
+    TERRA_COMMON_TRACING_SAMPLINGRATE: 0.1
```

Comment:
Were we able to get some wider clarity on what the best practices for sampling rate are, across DSP?  That could be worth asking about in #dsp-engineering .

Reply:
Here's the suggestion from the [DSP tracing blog](https://broadworkbench.atlassian.net/wiki/spaces/DSPBLOG/blog/2023/12/13/2930082050/Tracing+Terra#Performance):

> Typically the dev environment has this set to 1 and production to some low fraction like 0.1.

So, I missed the dev caveat, so I'll switch this to 1 and we can use 0.1 in production. 
Given the recency of this blog, I think this is a good starting point. We can dig into this more when the OpenTelemetry ticket - https://broadworkbench.atlassian.net/browse/DR-3456. I'll make sure this is mentioned in this ticket.